dotnet/src/SemanticKernel.UnitTests/Functions/FunctionFromMethodTests.cs

```diff
@@ -107,7 +108,7 @@ public async Task InvokeStreamingAsyncInvokingSkippingShouldRenderNoChunksAsync(
         var chunkCount = 0;
 
         // Act
-        await foreach (var chunk in sut.InvokeStreamingAsync<StreamingContent>(kernel, kernel.CreateNewContext()))
+        await foreach (var chunk in sut.InvokeStreamingAsync<StreamingContent>(kernel, new ContextVariables()))
```

Comment:
With your change, this method can have this parameter as optional, defaulting to new ContextVariables()

Reply:
I'll do it in one of the PRs when replacing parameters of ContextVariables type by KernelFunctionArguments or the one afterwards.